.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Bug Report
+about: Create a report to help us improve Telescope
+title: ''
+labels: kind/bug
```

Comment:
Should this be `type: bug`? to match what we have in our labels.  I'm  not sure.

Reply:
I tested type: bug, and I don't think it likes that formatting due to labels already using the semicolon.

Or I may be misunderstanding and you mean replacing the entire thing with type: bug?

Reply:
I got it, it needed to be enclosed in quotes in order to not be read as an additional specifier.